src/components/ShowAllJobs/showAllJobs.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import ShowAllJobs from './ShowAllJobs';
+import axios from 'axios';
+jest.mock('axios');
+const mockResponse = {
+  data: [
+    {
+      id: 1,
+      title: 'Job 1',
+      description: 'Job description 1',
+    },
+    {
+      id: 2,
+      title: 'Job 2',
+      description: 'Job description 2',
+    },
+  ],
+};
+describe('ShowAllJobs component', () => {
+  test('displays the correct number of jobs', async () => {  
+    const axiosGetMock = jest.spyOn(axios, 'get');
+    axiosGetMock.mockResolvedValueOnce({ data: mockResponse });
+    const { getByText } = render(<ShowAllJobs />);
+    await waitFor(() => {
+      expect(axiosGetMock).toHaveBeenCalledTimes(1);
+      expect(getByText('2 It jobs in viet nam')).toBeInTheDocument();
```

Comment:
should be using `expect(getByText(`${mockResponse.length} It jobs in viet nam`)).toBeInTheDocument();`

Reply:
fixed